rules/integrations/azure/initial_access_consent_grant_attack_via_azure_registered_application.toml

```diff
@@ -18,16 +18,70 @@ license = "Elastic License v2"
+  - Click on the `Review permissions` button on the `Permissions` blade of the application.
+  - An app should require only permissions related to the app's purpose. If that's not the case, the app might be risky.
+  - Apps that require high privileges or admin consent are more likely to be risky.
+- Investigate the app and the publisher. The following characteristics can indicate suspicious apps:
+  -  A low number of downloads.
+  -  Low rating or score or bad comments.
+  -  Apps with a suspicious publisher or website.
+  -  Apps whose last update is not recent. This might indicate an app that is no longer supported.
+- Export and examine the [Oauth app auditing](https://docs.microsoft.com/en-us/defender-cloud-apps/manage-app-permissions#oauth-app-auditing).
+To identify users affected.
+
+### False positive analysis
+
+- This mechanism can be used legitimately. Malicious applications abuse the same workflow that powers legitimate apps.
+Thus, analysts must review every app consent to ensure that only desired apps are granted access.
+
+### Response and remediation
+
+- Initiate the incident response process based on the outcome of the triage.
+- Identify the possible impact of the incident and prioritize accordingly; the following actions can help you gain context:
+    - Identify the account role in the cloud environment.
+    - Identify the services or servers involved criticality.
+    - Work with your IT team to identify the impact on users.
+    - Identify if the attacker is moving laterally and compromising other accounts, servers, or services.
+    - Identify if there are any regulatory or legal ramifications related to this activity.
+- Disable the malicious application to stop user access and the application access to your data.
+- Revoke the application Oauth consent grant. The `Remove-AzureADOAuth2PermissionGrant` cmdlet can be used to complete
+this task.
+- Remove the service principal application role assignment. The `Remove-AzureADServiceAppRoleAssignment` cmdlet can be
+used to complete this task.
+- Revoke the refresh token for all users assigned to the application. Azure provides a [playbook](https://github.com/Azure/Azure-Sentinel/tree/master/Playbooks/Revoke-AADSignInSessions) for this task.
+- [Report](https://docs.microsoft.com/en-us/defender-cloud-apps/manage-app-permissions#send-feedback) the application as malicious to Microsoft.
+- Investigate credential exposure on systems compromised or used by the attacker to ensure all compromised accounts are
+identified. Reset passwords or delete API keys as needed to revoke the attacker's access to the environment. Work with
+your IT teams to minimize the impact on the business operation during these actions.
+- Investigate potential data compromised from the user's email and file sharing services. Activate your Data Loss incident
+response playbook.
+- Disable the permission for a user to set consent permission on their behalf.
```

Comment:
```suggestion
- Prevent users from setting consent permissions without an administrator:
```

Reply:
The initial sentence seems to make more sense in this context